app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -1,82 +1,181 @@
 package com.wire.android.ui.home.conversationslist
 
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Divider
```

Comment:
This is what I was worried about - using old material elements instead of material3 ones - can we just use old material only when it's absolutely necessary?

`CircularProgressIndicator` is right now being added to the material3, soon it'll be available, until then we can use the one that I created - it's in the project already

Reply:
Actually, I think you can try and update material3 to `1.0.0-alpha04` - it should already have both `CircularProgressIndicator` and `Divider`